<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F7F7F7

</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1349</width>
     <height>758</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-image: url(D:/Program Files (x86)/Study/NEW_JumpTest_1.0/public/resources/pose_analysis.png);
    background-repeat: no-repeat;
    background-position: center;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ca_photo_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>照片</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sno_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; color: white;font-size: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; color: white;font-size: 50px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_match_posture">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>231</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_reference_posture">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>201</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableAngles">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>226</y>
     <width>433</width>
     <height>273</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textScoring">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>325</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: transparent;
    border: none;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pageLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string>第 1/3 页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevPageBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPageBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 700 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>参考标准</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textSuggestions">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>600</y>
     <width>1241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: transparent;
    border: none;
    padding: 5px;
    color: white;
    font-size: 20px;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QTextBrowser" name="labelkeyScore">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>120</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {
    background: transparent;
    border: none;
}

QTextBrowser[objectName=&quot;labelkeyScore&quot;] .phase-score {
    font-family: &quot;Arial&quot;;
    font-size: 48px;
    font-weight: bold;
    color: white;
}

QTextBrowser[objectName=&quot;labelkeyScore&quot;] .evaluation {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 14px;
    color: #666666;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevPoseBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton  {
background: transparent; border: none
 
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPoseBtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton  {
background: transparent; border: none
 
}



</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>tableWidget</zorder>
  <zorder>Ca_photo_2</zorder>
  <zorder>sno_label</zorder>
  <zorder>name_label</zorder>
  <zorder>label_match_posture</zorder>
  <zorder>label_reference_posture</zorder>
  <zorder>textScoring</zorder>
  <zorder>pageLabel</zorder>
  <zorder>prevPageBtn</zorder>
  <zorder>nextPageBtn</zorder>
  <zorder>label</zorder>
  <zorder>textSuggestions</zorder>
  <zorder>labelkeyScore</zorder>
  <zorder>tableAngles</zorder>
  <zorder>prevPoseBtn</zorder>
  <zorder>nextPoseBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
